default-config.json

```diff
@@ -4,7 +4,7 @@
     "stop": "Cadence stop",
     "help": "Cadence help",
     "nowplaying": "Cadence now playing",
-    "search": "Cadence search ",
+    "search": "Cadence search <query>",
```

Comment:
Side-note: This diff, as it stands, will change the trigger for `Cadence search` to include the literal `<query>`. IE these values serve dual duties as the left side of helptext and as the text CadenceBot looks for to trigger the command.

I won't demand you fix it though - I like where you're going with it, so I'll put in the legwork by myself to fix this to display and work properly between when I merge to master and when I push it to prod.